Koronavilkku/Koronavilkku/ViewController/MainViewController.swift

```diff
@@ -208,10 +208,12 @@ class MainViewController: UIViewController {
     }
 
     @objc private func infoButtonTapped() {
+        HapticEngine.selection.selectionChanged()
         showGuide()
     }
     
     @objc private func logoImageTapped() {
+        HapticEngine.result.notificationOccurred(.success)
```

Comment:
I think playing the `.success` fanfare for just selecting the logo and opening THL's website is a bit too much maybe 😄

And generally I might suggest to keep the haptic feedback only for clear selection, success, failure, and warning situations. Taking a cue from how Apple does it, haptic feedback is never triggered e.g. from simple navigation actions.

Reply:
Fair enough. Will remove that one.